sdk/core/azure-core/src/main/java/com/azure/core/http/policy/HttpLoggingPolicy.java

```diff
@@ -196,9 +191,11 @@ private String getAllowedQueryString(String queryString) {
                 bodySize = contentLengthString + "-byte";
             }
             HttpLogDetailLevel httpLogLevel = httpLogOptions.getLogLevel();
-            // HttpResponseStatus responseStatus = HttpResponseStatus.valueOf(response.statusCode());
             if (httpLogLevel.shouldLogUrl()) {
-                logger.info("<-- {} {} ({} ms, {} body)", response.getStatusCode(), url, tookMs, bodySize);
+                UrlBuilder requestUrl = UrlBuilder.parse(url);
+                requestUrl.setQuery(getAllowedQueryString(url.getQuery()));
+                logger.info("<-- {} {} ({} ms, {} body)", response.getStatusCode(), requestUrl.toString(), tookMs,
+                    bodySize);
```

Comment:
L85-L87 and this segment can be de-duped.

Reply:
Leaving as is to match the commit.